app/controllers/api/v1/users_controller.rb

```diff
@@ -25,22 +25,45 @@ def update
 
   def show
     user = User.find(params[:id])
-    # profile_picture = rails_blob_path(user.profile_picture)
     render json: UserSerializer.new(user)
   end
 
   def profile_picture
-    user = User.find_by(id: params[:id])
-    if user&.profile_picture&.attached?
-      redirect_to rails_blob_url(user.profile_picture)
+    user = User.find(params[:user_id])
+    if params[:profile_picture].present?
+      user.profile_picture.purge
+      user.profile_picture.attach(params[:profile_picture])
+      render json: UserSerializer.new(user)
     else
-      head :not_found
+      render json: {
+        error: 'No image file detected'
+      }, status: 400
     end
   end
 
+  # def profile_picture
+  #   user = User.find(params[:user_id])
+  #   attach_picture(user)
+  #   render json: UserSerializer.new(user)
+  # end
+
```

Comment:
left these commented out for Noah's reference.

Reply:
thanks!